doc/nrf/gs_programming.rst

```diff
@@ -86,8 +86,25 @@ Complete the following steps to build |NCS| projects with SES after :ref:`instal
 
    .. build_SES_projimport_start
 
-4. Click :guilabel:`OK` to add the project to SES.
-   You can now work with the project in the IDE.
+4. Click :guilabel:`OK` to add the project to SES. If you are Importing the project for the first time, you need to request and activate a license.
```

Comment:
A bit confused here. In line 43 we mention `Open NCS project`, but in this line suddenly Importing appears, suggesting that some other option is being used or could be used. This can be understood as using "File > Import Project".